### PR TITLE
Ome services clean up

### PR DESCRIPTION
**Pull request by jburel**
In this PR, we fix the remaining javadoc warning under blitz.
Check that building the blitz javadoc issues no warnings:
e.g.
```
Constructing Javadoc information...
Standard Doclet version 1.8.0_51
Building tree for all the packages and classes...
Building index for all the packages and classes...
Building index for all classes...
Building jar: /Users/jmarie/Documents/git-repo/openmicroscopy/components/blitz/target/blitz-javadoc.jar
```

### CODE REVIEW

File: components/blitz/src/ome/services/blitz/measurements/Helper.java [hunk 1]
```diff
@@ -31,8 +29,8 @@
      * Factory method to generate Helper instances from the LSID of a given
      * column.
      * 
-     * @param lsid
-     * @return
+     * @param lsid The lsid
+     * @return See above.
```
Comment (joshmoore) [inline components/blitz/src/ome/services/blitz/measurements/Helper.java]: These were removed elsewhere. Was there a guideline discussed before the cleaning started?

File: components/blitz/src/ome/services/blitz/repo/InternalRepositoryI.java [hunk 2]
```diff
@@ -20,7 +20,7 @@
 /**
  * Standalone repository service.
  *
- * @DEV.TODO Better named "StandaloneRepositoryI"
+ * TODO Better named "StandaloneRepositoryI"
```
Comment (joshmoore) [inline components/blitz/src/ome/services/blitz/repo/InternalRepositoryI.java]: Think @mtbc moved these to `//`-style comments so they don't show up in the javadoc.

File: components/blitz/src/ome/services/blitz/measurements/Helper.java (hunk 1 above)
Comment (mtbc) [inline components/blitz/src/ome/services/blitz/measurements/Helper.java]: Not as far as I know.

**Comment by sbesson** [comment]
From the [build standpoint](https://ci.openmicroscopy.org/job/OMERO-DEV-merge-build/ICE=3.5,jdk=8_LATEST,label=octopus/84/consoleText), 

```
Constructing Javadoc information...
Standard Doclet version 1.8.0_51
Building tree for all the packages and classes...
Building index for all the packages and classes...
Building index for all classes...
Building jar: /opt/hudson/workspace/OMERO-DEV-merge-build/src/components/blitz/target/blitz-javadoc.jar
```

This leaves the question of the Javadocs semantics (`//TODO` vs `TODO`, no `@param` vs stub `@param`). Should we get to this as part of this PR or finish up the build cleanup and then decide on Javadoc guidelines?

**Comment by jburel** [comment]
Probably better to finish the cleaning
Almost done with server component

File: components/blitz/src/ome/services/blitz/measurements/Helper.java (hunk 1 above)
Comment (jburel) [inline components/blitz/src/ome/services/blitz/measurements/Helper.java]: similar to tag work, guideline evolved as we were going 
and also due to the style of people "cleaning" the doc
Another round will be required for header

**Comment by joshmoore** [comment]
> similar to tag work, guideline evolved as we were going 
> and also due to the style of people "cleaning" the doc
> Another round will be required for header

Fair enough. Then maybe keep this one consistent and we'll review one last time once we've compared & chosen styles.

**Comment by sbesson** [comment]
From https://ci.openmicroscopy.org/view/DEV/job/OMERO-DEV-merge-build/88/ICE=3.5,jdk=8_LATEST,label=octopus/consoleText

```
Constructing Javadoc information...
Standard Doclet version 1.8.0_51
Building tree for all the packages and classes...
Building index for all the packages and classes...
Building index for all classes...
Building jar: /opt/hudson/workspace/OMERO-DEV-merge-build/src/components/blitz/target/blitz-javadoc.jar
```

and the TODO items are no longer present in the Javadocs. Ready to merge unless there is an extra commit scheduled.

**Comment by sbesson** [comment]
Assuming no other comment, additional Blitz cleanup (headers...) can be done in another PR. Merging.

**Comment by jburel** [comment]
--no-rebase

**Pull request merged**